scene/2d/cpu_particles_2d.h

```diff
@@ -179,6 +180,8 @@ class CPUParticles2D : public Node2D {
 
 	Vector2 gravity = Vector2(0, 980);
 
+	Ref<RandomNumberGenerator> _rng;
```

Comment:
```suggestion
	Ref<RandomNumberGenerator> rng;
```
We don't underscore private variables.